Dockerfile

```diff
@@ -1,13 +1,22 @@
 FROM alpine
 MAINTAINER Rémi Alvergnat <toilal.dev@gmail.com>
 
-RUN apk add --update python py-pip ca-certificates && rm -rf /var/cache/apk/*
+RUN apk add --update python3 ca-certificates nodejs build-base python3-dev libffi-dev openssl-dev unzip && \
+    rm -rf /var/cache/apk/*
 
 ADD . /opt/flexget
 WORKDIR /opt/flexget
 
-RUN pip install paver
-RUN pip install -e .
+RUN pip3 install paver
+RUN pip3 install -e .
+RUN pip3 install transmissionrpc
```

Comment:
Hmm, should we include other optional deps for common plugins, like deluge-client?

Reply:
Yes make sense. Do you have a corresponding dep list?

Reply:
We don't. I guess let's just put deluge-client in here and we can add other things later if we deem necessary.

Reply:
OK. Added the dep.